deploy/01-deploy-fund-me.ts

```diff
@@ -1,40 +1,46 @@
-import { HardhatRuntimeEnvironment } from "hardhat/types"
-import { DeployFunction } from "hardhat-deploy/types"
-import verify from "../utils/verify"
-import { networkConfig, developmentChains } from "../helper-hardhat-config"
-
-const deployFundMe: DeployFunction = async function (
-  hre: HardhatRuntimeEnvironment
-) {
-  // @ts-ignore
-  const { getNamedAccounts, deployments, network } = hre
-  const { deploy, log } = deployments
-  const { deployer } = await getNamedAccounts()
-  const chainId: number = network.config.chainId!
-
-  let ethUsdPriceFeedAddress: string
-  if (chainId == 31337) {
-    const ethUsdAggregator = await deployments.get("MockV3Aggregator")
-    ethUsdPriceFeedAddress = ethUsdAggregator.address
-  } else {
-    ethUsdPriceFeedAddress = networkConfig[network.name].ethUsdPriceFeed!
-  }
-  log("----------------------------------------------------")
-  log("Deploying FundMe and waiting for confirmations...")
-  const fundMe = await deploy("FundMe", {
-    from: deployer,
-    args: [ethUsdPriceFeedAddress],
-    log: true,
-    // we need to wait if on a live network so we can verify properly
-    waitConfirmations: networkConfig[network.name].blockConfirmations || 0,
-  })
-  log(`FundMe deployed at ${fundMe.address}`)
-  if (
-    !developmentChains.includes(network.name) &&
-    process.env.ETHERSCAN_API_KEY
-  ) {
-    await verify(fundMe.address, [ethUsdPriceFeedAddress])
-  }
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { DeployFunction } from 'hardhat-deploy/types'
+import { network } from 'hardhat'
+
+import { developmentChains, networkConfig } from '../helper-hardhat-config'
+import verify from '../utils/verify'
+
+const deployFunction: DeployFunction = async function ({
+    getNamedAccounts,
+    deployments
+}: HardhatRuntimeEnvironment) {
+    const { deploy, log, get } = deployments
+    const { deployer } = await getNamedAccounts()
+    const chainId = network.config.chainId!
+
+    let ethUsdPriceFeedAddress
+
+    if (developmentChains.includes(network.name)) {
+        const ethUsdAggregator = await get('MockV3Aggregator')
+        ethUsdPriceFeedAddress = ethUsdAggregator.address
+    } else {
+        ethUsdPriceFeedAddress = networkConfig[chainId].ethUsdPriceFeed
+    }
+
+    const args = [ethUsdPriceFeedAddress]
+
+    const fundMe = await deploy('FundMe', {
+        from: deployer,
+        args,
+        log: true,
+        waitConfirmations: 1
+    })
+
+    if (
+        !developmentChains.includes(network.name) &&
+        process.env.ETHERSCAN_API_KEY
+    ) {
+        await verify(fundMe.address, args)
+    }
+
+    log('---------------------------')
 }
-export default deployFundMe
-deployFundMe.tags = ["all", "fundMe"]
+
+export default deployFunction
```

Comment:
This should be `deployFundMe`, otherwise when people import `deployFunction`, it's hard to tell which one they imported!